<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="readyData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dataSend">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单次发</string>
    </property>
   </widget>
   <widget class="QLabel" name="systemTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系统时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="systemTimeShow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2018.9.22</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerInterval">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>定时器间隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerUnit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="runTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>已走时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="protocolData">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看协议表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readCanData">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>读取Can数据</string>
    </property>
   </widget>
   <widget class="QTableView" name="canDataTable">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setOrReset">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rebot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重启</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shutDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="other">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>其它</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerIntervalShow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="runTimeShow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataTimerSend">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="readyTimerData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopTimer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondSend">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data1_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="list_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secondSend_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data1_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="list_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secondSend_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data1_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="list_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secondSend_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data1_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="list_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>第</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time1_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secondSend_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataSend_list">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生效并开始</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipAddress">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="connectPort">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startConnect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始连接</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
